docs/sphinx/users/fiji/index.txt

```diff
@@ -23,3 +23,81 @@ checks for updates every time it is launched, so you will always be
 notified when new versions of Bio-Formats (or any other bundled plugin)
 are available.
 
+Using Bio-Formats daily builds
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Fiji currently shipping with the 5.1.x release versions of Bio-Formats.
+However, if you have encountered a bug which has been fixed by the Bio-Formats
+team but not yet released, you can use the Bio-Formats update site to
+access the daily build as described in the
+`Fiji documentation <http://fiji.sc/Bio-Formats#Daily_builds>`_.
+
+.. warning:: These builds are **not yet released** and should be considered
+    **beta** in quality. In particular, you should **avoid exporting data
+    using the Bio-Formats Exporter** in case you write incompatible files
```

Comment:
The only bit here that I'm particularly worried about is the bit about incompatible files:
> In particular, you should **avoid exporting data
> using the Bio-Formats Exporter** in case you write incompatible files

 since that only holds for DEV releases: